docs/release_notes.rst

```diff
@@ -35,6 +35,7 @@ New Features
 * Allow for hanging indent when documenting args in Google style. (#449)
 * Add support for `property_decorators` config to ignore D401.
 * Add support for Python 3.10 (#554).
+* Add `ignore-self-only-init` config (#560).
```

Comment:
Can we move this to the appropriate version section?

Reply:
done